crates/rhyolite_bevy/src/lib.rs

```diff
@@ -100,6 +100,17 @@ impl Plugin for RenderPlugin {
                 .iter()
                 .map(|a| a.as_ptr())
                 .collect();
+
+            let event_loop: &winit::event_loop::EventLoop<()> = app.world.get_non_send_resource().expect("Your display are not supported");
```

Comment:
If we couldn't get the event loop from the world, it probably means that the RenderPlugin was inserted after WinitPlugin. I would just emit a warning with `tracing::warn!()` and skip adding the extension here.